src/onegov/org/path.py

```diff
@@ -458,6 +459,51 @@ def get_occurrences(app, request, page=0, range=None, start=None, end=None,
     )
 
 
+class tags(etree.ElementBase):
+    """
+    Custom class as 'tag' is a member of class Element and cannot be
+    used as tag name.
+    """
+    def __init__(self, tags=()):
+        super().__init__()
+        self.tag = 'tags'
+
+        for t in tags:
+            tag = etree.Element('tag')
+            tag.text = t
+            self.append(tag)
+
+
+@OrgApp.view(model=OccurrenceCollection, name='xml')
+def events_export_xml(app, request):
+    xml = '<events></events>'
+    root = objectify.fromstring(xml)
+
+    for ev in EventCollection(app.session).query().all():
+        event = objectify.Element('event')
```

Comment:
Please also comment in the docstring off the new collection function the format or link to the definition (I guess anthrazit? Mabybe even name the function `as_antrazit_xml`?), so that nobody comes up with the bad idea to change things here without checking the format first...